core/domain/stats_jobs_continuous_test.py

```diff
@@ -262,6 +307,17 @@ def test_answers_across_multiple_exploration_versions(self):
                     'state_name': second_state_name,
                     'property_name': exp_domain.STATE_PROPERTY_INTERACTION_ID,
                     'new_value': 'MultipleChoiceInput',
+                }), exp_domain.ExplorationChange({
```

Comment:
Why are the changes in this file necessary?

Reply:
did you remember, I have do try an exception in the codebase to fix tests?

So the point is if we use interaction like ```Multiplechoise, ItemSelection, DragAndDropOut ```, then for this interaction we have choice and value which are custom args of interaction. But since we were just changing interaction id and not adding custom_asgs for this type of interactions which leads to failure of tests.